### PR TITLE
Created first react component

### PR DESCRIPTION
**Pull request by gosupark27**
Creating my first pull request for resolving issue #1 @thatzacdavis

### CODE REVIEW

**Comment by gosupark27** [comment]
Used grid layout to create a 2x3 grid for the textboxes, created state variables for the rest of the form fields and added a create ticket button.

**Comment by thatzacdavis** [comment]
Make sure you add `/node_modules` to your `.gitignore`

File: api/routes/ticket.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+//Save Ticket 
+router.put('/', async (req,res) => {
+    console.log('Got body:', req.body);
+    res.sendStatus(200);
+
+    const result = await Ticket.create(req.body);
+    console.log(result);
+
+
```
Comment (thatzacdavis) [inline api/routes/ticket.js]:
```suggestion
```
Extra whitespace tends to make files longer with no real benefits.

File: api/routes/ticket.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+var express = require('express');
+var router = express.Router();
+var mongoose = require('mongoose');
+var Ticket = require('../models/Ticket');
+
+//Save Ticket
```
Comment (thatzacdavis) [inline api/routes/ticket.js]:
```suggestion
// Save Ticket 
```

Generally, there is a space before the beginning of a comment.

File: client/src/component/ticketForm.js [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        //Endpoint to express server 
+        const url = 'http://localhost:5000/testAPI';
+        
+        const ticketData = {
+            firstName, lastName, phone, pickUpDate, ticketNumber, dropDate
+        }
+        console.log(ticketData);
+
+        //Use post or put request?
+        axios.put(url, ticketData)
```
Comment (thatzacdavis) [inline client/src/component/ticketForm.js:25]: https://www.w3schools.com/tags/ref_httpmethods.asp#:~:text=PUT%20is%20used%20to%20send,that%20PUT%20requests%20are%20idempotent.&text=In%20contrast%2C%20calling%20a%20POST,the%20same%20resource%20multiple%20times.

So this technically depends on how we build the API.

File: client/src/component/ticketForm.js [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+    const [firstName, setFirstName] = useState('')
+    const [lastName, setLastName] = useState('')
+    const [phone, setPhone] = useState('')
+    const [pickUpDate, setPickUpDate] = useState('')
+    const [ticketNumber, setTicketNumber] = useState('')
+    const [dropDate, setDropDate] = useState('')
+    // const handleChange = (event) => {
+    //     setFirstName(event.target.value);
+    //     console.log(event.target);
+    //   };
```
Comment (thatzacdavis) [inline client/src/component/ticketForm.js]:
```suggestion
```

We don't need this anymore so there's no point in leaving it.

File: client/src/component/ticketForm.js [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+    //Axios Put request
```
Comment (thatzacdavis) [inline client/src/component/ticketForm.js]:
```suggestion
```

Generally, comments are there to provide context surround what the code is doing. We can see that the code is going to make a PUT w/ Axios, so comments like these aren't really necessary.

**Review by thatzacdavis — CHANGES_REQUESTED** [review]
Overall, this is a really good start. You've picked up how the component library works very well.

Let's get those comments and the whitespace cleaned up though.

Other than that and any comments I put inline, I'm not usually a fan of having a bunch of console logs, but maybe we can clean them up later.

**Pull request merged**